### DISCUSSION TITLE
Feature request: Ignore "upgrade" backups

### DISCUSSION THREAD

**Discussion by hockey6611** [lavinir/hassio-onedrive-backup#65] [Ideas]
It would be great if there was an option to ignore upgrade backups. The popular google drive backup adodn offers this option, and I've found it very useful.

**Comment by lavinir** [comment]
You mean the backups you are prompted to create before an upgrade? 
What would be the point - Those are optional so if you don't want them you can opt to skip them during upgrade right?
If you do want them but don't want them backed up to OneDrive you can ignore backups not created by the addon. (@hockey6611 )

**Reply by hockey6611** [reply]
Since home assistant defaults to make these backups, they need to be individually selected to not backup at each upgrade. So it is tedious to uncheck each time. Also, I would like to make these backups when upgrading to easily revert when there are issues. But I don't need to sync them.

 I also would like to sync other backups created outside of the add-on. So if upgrade backups can be excluded this would be more seamless. Currently the upgrade backups take up slots in the aloted number of backups for syncing to onedrive.

This feature is implemented in the hassio-google-drive-backup addon, which I used previously.

**Reply by lavinir** [reply]
I will add it to the backlog.

**Reply by 4homeassistant** [reply]
Great work @lavinir! I you have time I'd also be happy about adding this feature.

**Reply by lavinir** [reply]
this has been added in 2.1

**Reply by 4homeassistant** [reply]
Wow! Thank you so much 😊
